sophie-kaleba/assignment1/AnagramTest.java

```diff
@@ -0,0 +1,39 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
```

Comment:
These look like changes for Assignment1. It would have been better to have these as a separate pull request.

Reply:
Sorry I completely messed up between my 2 branches indeed